src/network/network.cpp

```diff
@@ -839,33 +863,16 @@ void Network::LoadGroups()
 	platform_get_user_directory(path, NULL);
 	strcat(path, "groups.json");
 
-	if (!platform_file_exists(path)) {
+	json_t * json = nullptr;
+	try {
+		json = Json::ReadFromFile(path);
+	} catch (const std::exception& e) {
```

Comment:
Use Exception instead which is inherits std::exception. This is to cope with MSVC / GCC, Clang differences.